<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpielFenster</class>
 <widget class="QDialog" name="SpielFenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>511</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QLabel" name="field2_1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="field1_0">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="field1_1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="field1_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="field2_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="field2_0">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="field0_1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="field0_0">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="field0_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Field_Icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
